### PR TITLE
Add the missing column name to event/sessions and admin/users table

### LINKED ISSUE #850: Add missing column names at the events/sessions and admin/users
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

No column name for the buttons.
Expected: Column names should be there.

### PR DESCRIPTION
**Pull request by sumedh123**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Fixes #850

### CODE REVIEW

**Comment by sumedh123** [comment]
@fossasia/open-event-frontend Please review.

**Comment by niranjan94** [comment]
Deployment link: https://pr-851-open-event-frontend-fossasia.surge.sh

File: app/controllers/admin/users/list.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ export default Controller.extend({
     },
     {
       template         : 'components/ui-table/cell/cell-buttons',
-      title            : '',
+      title            : 'Action',
```
Comment (CosmicCoder96) [inline app/controllers/admin/users/list.js]: make it `Actions`

**Comment by niranjan94** [comment]
Deployment link: https://pr-851-open-event-frontend-fossasia.surge.sh

**Comment by sumedh123** [comment]
@CosmicCoder96 Please review.

**Comment by niranjan94** [comment]
Deployment link: https://pr-851-open-event-frontend-fossasia.surge.sh

**Pull request merged**